addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -376,6 +288,40 @@ public void packetReceived(byte[] packet) {
                     if (msg.subType == SubType.RESPONSE) {
                         logger.debug("RFXCOM transceiver/receiver type: {}, hw version: {}.{}, fw version: {}",
                                 msg.transceiverType, msg.hardwareVersion1, msg.hardwareVersion2, msg.firmwareVersion);
+
+                        if (msg.command == Commands.GET_STATUS) {
+                            if (configuration.ignoreConfig) {
+                                logger.debug("Ignoring transceiver configuration");
+                            } else {
+                                byte[] setMode = new byte[0];
+
+                                try {
+                                    if (configuration.setMode != null && !configuration.setMode.isEmpty()) {
```

Comment:
I don't see this used anywhere else in the rfxcom binding. It seems a bit of an overhead just to avoid one comparison to null.